SymbolsGenerator/Sources/SymbolsGenerator/Resources/symbol_previews.txt

```diff
@@ -3,4 +3,4 @@
 // Updated on January 21 2022
```

Comment:
Nit: The file headers need to be updated for some files in the `Resources` folder, specifying the `SF Symbols` version and update date.